collectors/osd.go

```diff
@@ -127,6 +131,11 @@ type OSDCollector struct {
 
 	// OSDObjectsBackfilled displays average number of objects backfilled in an OSD
 	OSDObjectsBackfilled *prometheus.CounterVec
+
+	// OldestInactivePG gives us the amount of time that the oldest inactive PG
+	// has been inactive for.  This is useful in rolling peering storms, or if
+	// you're cancelling a lot of backfill.  See `pgremapper` for more :)
```

Comment:
I would reword the `This is useful` sentence to something like: `This is useful to discern between rolling peering (such as when issuing a bunch of upmaps or weight changes) and a single PG stuck peering, for example.`